test/NewBankTest.java

```diff
@@ -438,5 +438,36 @@ public void getHelp() {
                 "PAY <Amount> <Debit account> <Credit account>\tPays funds from one account to another account, which may be held by another customer\n" +
                 "HELP\t\t\tShows this menu\n\n", result);
     }
```

Comment:
The negative cases should also be tested

Reply:
Thank you. Will be optimised ;-).